app/views/dashboard/_node_meta.html.erb

```diff
@@ -1,6 +1,9 @@
 <% unless node.is_a?(Comment) || node.is_a?(Revision) || node.type == 'page' %>
   <%= t('dashboard.dashboard.by') %> <a href="/profile/<%= node.author.name %>"><%= node.author.name %></a> 
-<% end %>
+  <% if node.has_power_tag('with') %>with <% node.coauthors.each_with_index do |coauthor,i| %>
+              <a href="/profile/<%= coauthor.username %>">
+                <%= coauthor.username%></a>
+              <%= ',' if i < node.coauthors.length-1 %><% end %><% end %>
```

Comment:
Actually i think some clearer indentation formatting could help here; we use two spaces (no tabs) as a convention, and if you could have the `<% end %>` on its own line, it may be easier to read and debug. It does look like, from the error in the Travis tests, you're missing one of the `<% end %>`s! 

Hope that helps!